dali/test/python/test_pipeline.py

```diff
@@ -600,6 +600,67 @@ def define_graph(self):
             np_transposed = images[b].transpose((2, 0, 1))
             np_transposed = np.ascontiguousarray(np_transposed)
             assert(np.array_equal(np_transposed, images_transposed[b]))
+def test_equal_nvJPEGDecoderCrop_nvJPEGDecoder():
```

Comment:
```suggestion

def test_equal_nvJPEGDecoderCrop_nvJPEGDecoder():
```
leave one empty line before the start of the test